<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="15" failures="4" disabled="0" errors="0" timestamp="2018-11-08T06:01:09" time="0.918" name="AllTests">
  <testsuite name="TestEnv_Reliable" tests="2" failures="2" disabled="0" errors="0" time="0.189">
    <testcase name="TestAccept_Backlog1" status="run" time="0.12" classname="TestEnv_Reliable">
      <failure message="testhandshake.cpp:97&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 1&#x0A;To be equal to: expected_accept&#x0A;      Which is: 3" type=""><![CDATA[testhandshake.cpp:97
      Expected: (int)client_sockets.size()
      Which is: 1
To be equal to: expected_accept
      Which is: 3]]></failure>
      <failure message="testhandshake.cpp:179&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 0&#x0A;To be equal to: expected_connect&#x0A;      Which is: 1" type=""><![CDATA[testhandshake.cpp:179
      Expected: (int)client_sockets.size()
      Which is: 0
To be equal to: expected_connect
      Which is: 1]]></failure>
      <failure message="testhandshake.cpp:179&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 0&#x0A;To be equal to: expected_connect&#x0A;      Which is: 1" type=""><![CDATA[testhandshake.cpp:179
      Expected: (int)client_sockets.size()
      Which is: 0
To be equal to: expected_connect
      Which is: 1]]></failure>
    </testcase>
    <testcase name="TestAccept_Backlog2" status="run" time="0.069" classname="TestEnv_Reliable">
      <failure message="testhandshake.cpp:97&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 5&#x0A;To be equal to: expected_accept&#x0A;      Which is: 6" type=""><![CDATA[testhandshake.cpp:97
      Expected: (int)client_sockets.size()
      Which is: 5
To be equal to: expected_accept
      Which is: 6]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="TestEnv_Any" tests="13" failures="2" disabled="0" errors="0" time="0.729">
    <testcase name="TestClose_Connect_CloseFirst" status="run" time="0.003" classname="TestEnv_Any" />
    <testcase name="TestClose_Connect_CloseLater" status="run" time="0.002" classname="TestEnv_Any" />
    <testcase name="TestClose_Connect_CloseSimultaneous" status="run" time="0.003" classname="TestEnv_Any" />
    <testcase name="TestClose_Accept_CloseLater" status="run" time="0.025" classname="TestEnv_Any" />
    <testcase name="TestClose_Accept_CloseFirst" status="run" time="0.028" classname="TestEnv_Any" />
    <testcase name="TestClose_Accept_CloseSimultaneous" status="run" time="0.034" classname="TestEnv_Any" />
    <testcase name="TestAccept_BeforeAccept" status="run" time="0.029" classname="TestEnv_Any" />
    <testcase name="TestAccept_AfterAccept" status="run" time="0.028" classname="TestEnv_Any" />
    <testcase name="TestAccept_MultipleInterface1" status="run" time="0.307" classname="TestEnv_Any">
      <failure message="testhandshake.cpp:78&#x0A;      Expected: client_addr.sin_family&#x0A;      Which is: &apos;\0&apos;&#x0A;To be equal to: 2" type=""><![CDATA[testhandshake.cpp:78
      Expected: client_addr.sin_family
      Which is: '\0'
To be equal to: 2]]></failure>
      <failure message="testhandshake.cpp:109&#x0A;      Expected: same_count&#x0A;      Which is: 2&#x0A;To be equal to: 1" type=""><![CDATA[testhandshake.cpp:109
      Expected: same_count
      Which is: 2
To be equal to: 1]]></failure>
      <failure message="testhandshake.cpp:109&#x0A;      Expected: same_count&#x0A;      Which is: 2&#x0A;To be equal to: 1" type=""><![CDATA[testhandshake.cpp:109
      Expected: same_count
      Which is: 2
To be equal to: 1]]></failure>
      <failure message="testhandshake.cpp:97&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 0&#x0A;To be equal to: expected_accept&#x0A;      Which is: 5" type=""><![CDATA[testhandshake.cpp:97
      Expected: (int)client_sockets.size()
      Which is: 0
To be equal to: expected_accept
      Which is: 5]]></failure>
    </testcase>
    <testcase name="TestAccept_MultipleInterface2" status="run" time="0.257" classname="TestEnv_Any">
      <failure message="testhandshake.cpp:87&#x0A;Value of: (addr.sin_addr.s_addr == 0) || (addr.sin_addr.s_addr == temp_addr.sin_addr.s_addr)&#x0A;  Actual: false&#x0A;Expected: true" type=""><![CDATA[testhandshake.cpp:87
Value of: (addr.sin_addr.s_addr == 0) || (addr.sin_addr.s_addr == temp_addr.sin_addr.s_addr)
  Actual: false
Expected: true]]></failure>
      <failure message="testhandshake.cpp:87&#x0A;Value of: (addr.sin_addr.s_addr == 0) || (addr.sin_addr.s_addr == temp_addr.sin_addr.s_addr)&#x0A;  Actual: false&#x0A;Expected: true" type=""><![CDATA[testhandshake.cpp:87
Value of: (addr.sin_addr.s_addr == 0) || (addr.sin_addr.s_addr == temp_addr.sin_addr.s_addr)
  Actual: false
Expected: true]]></failure>
      <failure message="testhandshake.cpp:97&#x0A;      Expected: (int)client_sockets.size()&#x0A;      Which is: 2&#x0A;To be equal to: expected_accept&#x0A;      Which is: 4" type=""><![CDATA[testhandshake.cpp:97
      Expected: (int)client_sockets.size()
      Which is: 2
To be equal to: expected_accept
      Which is: 4]]></failure>
    </testcase>
    <testcase name="TestConnect_BeforeAccept" status="run" time="0.003" classname="TestEnv_Any" />
    <testcase name="TestConnect_AfterAccept" status="run" time="0.002" classname="TestEnv_Any" />
    <testcase name="TestConnect_SimultaneousConnect" status="run" time="0.002" classname="TestEnv_Any" />
  </testsuite>
</testsuites>
